subsys/logging/log_backend_adsp.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log_backend.h>
+#include <logging/log_core.h>
+#include <logging/log_msg.h>
+#include <logging/log_output.h>
+#include <sys/ring_buffer.h>
+#include <device.h>
+#include <assert.h>
+#include <soc.h>
+
+#define BUF_SIZE 64
+
+static struct ring_buf ringbuf;
+
+static u16_t magic = 0x55aa;
+static u16_t log_id;
```

Comment:
use only in `trace()`. Could be moved there.

Reply:
fixed